hazelcast/src/main/java/com/hazelcast/jet/JobStatusListener.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet;
+
+import java.util.UUID;
+
+/**
+ * Invoked upon job status change.
+ *
+ * @see Job#addStatusListener(JobStatusListener)
+ * @see Job#removeStatusListener(UUID)
+ * @since 5.3
+ */
+public interface JobStatusListener {
+    void jobStatusChanged(JobStatusEvent event);
```

Comment:
Missing javadoc.

Reply:
The Javadoc of the class says "invoked upon job status change," and the name of the method is "jobStatusChanged." I personally avoid comments that state obvious things.